src/main/g8/build.sbt

```diff
@@ -75,12 +63,6 @@ sparkCoreType := "m4.xlarge"
 sparkInstanceCount := 5
 sparkMasterPrice := Some(0.5)
 sparkCorePrice := Some(0.5)
-sparkEmrBootstrap := List(
-  BootstrapAction("Install GDAL + dependencies",
-                  "s3://geotrellis-test/usbuildings/bootstrap.sh",
-                  "s3://geotrellis-test/usbuildings",
-                  "v1.0")
```

Comment:
Ah, why was it removed? We don't need it anymore?

Reply:
I'd like to use geotrellis-deployments tagged release here. That was just something we published on a job, danger of it going stale and breaking at some point in the future.